apps/web/src/utils/vaidation.ts

```diff
@@ -0,0 +1,9 @@
+export const RegularEmailValidation = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
+
+export const MinimalLength = (t: any, d: any) => {
```

Comment:
Тут не нужно функцией, сделай просто константой 
```suggestion
export const MIN_PASS_LENGTH = 1
export const MAX_NAME_LENGTH = 30
```